src/Service/AllInBadges.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Maybe not in dir `Service` but new dir `Configuration`?

Reply:
@AlessandroMinoccheri this?

Reply:
@JellyBellyDev in my mind there are many solutions: `Dictionary`, `Enum` or `Configuration`.
I think that `Service` is not the right place for `AllInBadges` and `Badges` classes.
Do you think that `Configuration` could be the best solution now?

Reply:
Maybe I prefer `Dictionary`! 👍